cmd/api/api.go

```diff
@@ -63,15 +63,13 @@ func NewAPICmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "api",
 		Short: "Run the controller API",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			c := command{CLIOptions: config.GetCmdOpts()}
-
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
 			logrus.SetOutput(os.Stdout)
```

Comment:
(this should be `logrus.SetOutput(cmd.OutOrStdout())`)

Reply:
#2206